nautobot/core/ui/choices.py

```diff
@@ -18,7 +25,13 @@ class LayoutChoices(ChoiceSet):
 
 
 class SectionChoices(ChoiceSet):
-    """Sections of a Layout to assign panels to."""
+    """Sections of a Layout to assign panels to. Placement of panels is determined by [`LayoutChoices`]( set on Tab.layout
```

Comment:
```suggestion
    """Sections of a Layout to assign panels to.
    
    Placement of panels is determined by `LayoutChoices` set on `Tab.layout`.
```

Reply:
I wanted to add link there but apparently I forgot during the search for url. Should I remove link or just properly add it?

Reply:
Adding the link probably is desirable. :)